app/jobs/chat_job.rb

```diff
@@ -0,0 +1,295 @@
+class ChatJob < ApplicationJob
+  queue_as :default
+
+  def perform(chat_id, message_id)
+    chat = Chat.find(chat_id)
+    nil if chat.nil?
```

Comment:
This line won't ever be hit because `find` would raise and `ActiveRecord::RecordNotFound` error if there isn't a `Chat` for the `chat_id`.